AsyncDisplayKit/ASImageNode.mm

```diff
@@ -323,25 +458,27 @@ - (UIImage *)displayWithParameters:(id<NSObject> *)parameter isCancelled:(asdisp
   // Another option is to have ASDisplayNode+AsyncDisplay coordinate these cases, and share the decoded buffer.
   // Details tracked in https://github.com/facebook/AsyncDisplayKit/issues/1068
   
-  @synchronized(image) {
-    [image drawInRect:imageDrawRect];
+  @synchronized(key.image) {
+    [key.image drawInRect:key.imageDrawRect];
```

Comment:
In the future we could use associated objects to attach an NSLock to this image and use that instead, since it's allegedly so much faster http://perpendiculo.us/2009/09/synchronized-nslock-pthread-osspinlock-showdown-done-right/